core/src/main/kotlin/gui/MenuScreen.kt

```diff
@@ -26,56 +26,28 @@ class MenuScreen(private val game: gui.Game) : Screen {
     lateinit var title: Texture
     lateinit var checkerSprite: Sprite
     lateinit var titleSprite: Sprite
-    private lateinit var viewport: Viewport
-    private lateinit var camera: Camera
 
 
-    private val newGameButton = Button(Texture(GUIConstants.newGameButtonUpPath),
-            Texture(GUIConstants.newGameButtonDownPath),
-            GUIConstants.newGameButtonX, GUIConstants.newGameButtonY,
-            GUIConstants.newGameButtonWidth, GUIConstants.newGameButtonHeight)
+    private val newGameButton = Button(Texture(GUIConstants.newGameButton.upPath),
```

Comment:
You may use this function/constructor too

Reply:
Fixed